### PR TITLE
Inventory Class

### PR DESCRIPTION
**Pull request by CarrilloMario831**
Inventory Class

### CODE REVIEW

File: src/main/java/service/models/Inventory.java [hunk 1]
```diff
@@ -1,17 +1,141 @@
...
+    addItem(item);
+  }
+
+  /**
+   * Returns the UUID of the inventory.
+   *
+   * @return the UUID of the inventory
+   */
+  public UUID getId() {
+    return uuid;
```
Comment (sjimenez814) [inline src/main/java/service/models/Inventory.java]: Replace with `@Getter`

File: src/main/java/service/models/Inventory.java [hunk 2]
```diff
@@ -1,17 +1,141 @@
...
+    return inventoryName;
+  }
+
+  /**
+   * Sets the name of the inventory.
+   *
+   * @param inventoryName the new inventory name
+   */
+  public void setInventoryName(String inventoryName) {
+    this.inventoryName = inventoryName;
```
Comment (sjimenez814) [inline src/main/java/service/models/Inventory.java]: user `@Setter` and `@Getter` on fields that we can set/get. If all of them need getters or setters, then we can have 1 annotation at the beginning of the class

**Review by sjimenez814 — APPROVED** [review]
Code looks good. Just the Lombok annotations to remove unnecessary code.

File: src/main/java/service/models/Inventory.java (hunk 2 above)
Comment (CarrilloMario831) [reply src/main/java/service/models/Inventory.java]: done

File: src/main/java/service/models/Inventory.java (hunk 1 above)
Comment (CarrilloMario831) [reply src/main/java/service/models/Inventory.java]: done

**Review by dwr2118 — APPROVED** [review]

**Pull request merged**